sensor_msgs/Image rgb_image
sensor_msgs/Image depth_image
sensor_msgs/RegionOfInterest bounding_box
---
int32 num_keypoints
geometry_msgs/Point[] keypoints_camera_frame # The detected keypoint